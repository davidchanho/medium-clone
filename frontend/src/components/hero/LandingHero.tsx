import React from "react";
import { Button } from "react-bootstrap";

function LandingHero() {
  return (
    <div className='bg-yellow'>
      <h4>Where good ideas find you</h4>

      <p>Read and share new perspectives on just about any topic.</p>
      <p>
        Everyone's welcome. <a href="#!">Learn more.</a>
      </p>

      <Button>Get Started</Button>
    </div>
  );
}

export default LandingHero;
