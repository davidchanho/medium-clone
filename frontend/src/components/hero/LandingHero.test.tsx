import React from "react";
import renderer from "react-test-renderer";
import LandingHero from "./LandingHero";

describe("LandingHero", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<LandingHero />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
