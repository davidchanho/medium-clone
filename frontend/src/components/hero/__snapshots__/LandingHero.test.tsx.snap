// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingHero renders correctly 1`] = `
<div>
  <h4>
    Where good ideas find you
  </h4>
  <p>
    Read and share new perspectives on just about any topic.
  </p>
  <p>
    Everyone's welcome. 
    <a
      href="#!"
    >
      Learn more.
    </a>
  </p>
  <button
    className="btn btn-primary"
    disabled={false}
    type="button"
  >
    Get Started
  </button>
</div>
`;
